src/cli/main.py

```
@@ -250,6 +250,28 @@ def history(
     raise typer.Exit(code=code)
 
 
+@app.command()
+def recover(  # noqa: G3 (--journal is a read-only path; defaults to system state dir)
```

Ruff (0.15.11)

[warning] Invalid rule code in `# noqa`: G3

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)